src/frontend/src/player.ts

```diff
@@ -67,17 +67,19 @@ export class Player {
     Object.entries(this._ipcEvents).forEach(event => ipcMain.on(event[0], event[1]))
   }
 
-  initStore(): Store<IPlayerStore> {
+  initStore(playerConfig: IPlayerStore): void {
     const shouldPersist = !storeManager.exists('player')
-    const store = storeManager.createStore<IPlayerStore>('player', DEFAULT_PLAYER_STORE)
 
-    if (shouldPersist) {
-      store.persist()
-    } else if (store.get('version') !== app.getVersion()) {
-      store.reset(DEFAULT_PLAYER_STORE)
+    if (storeManager.exists('player')) {
+      const playerConfigMerged = mergeWith({}, this.flowrStore.data.player, DEFAULT_PLAYER_STORE, playerConfig, (a, b) => b === null || b === '' ? a : undefined)
+      this.store.bulkSet(playerConfigMerged)
     }
 
-    return store
+    if (shouldPersist) {
+      this.store.persist()
+    } else if (this.store.get('version') !== app.getVersion()) {
```

Comment:
This `else if` statement is not necessary anymore. It is now obsolete with the sync from flowr-admin